crates/types/src/calibration.rs

```diff
@@ -9,16 +9,11 @@ use crate::{camera_position::CameraPosition, cycle_time::CycleTime};
 #[derive(
     Copy, Clone, Debug, Serialize, Deserialize, PathSerialize, PathDeserialize, PathIntrospect,
 )]
-pub enum CalibrationCommand {
-    LookAt {
-        target: Point2<Ground>,
-        camera: CameraPosition,
-        dispatch_time: CycleTime,
-    },
-    Capture {
-        camera: CameraPosition,
-        dispatch_time: CycleTime,
-    },
+pub struct CalibrationCaptureCommand {
```

Comment:
Is it really a "capture" command?
I think the old name was better.

Reply:
Makes sense :D reverted it.